tuist/tuist
# Fix CI swiftlint checks

### Short description 📝

Run switlint with --strict flag, or warnings are ignored in CI

### How to test the changes locally 🧐

1. go on master
2. run `./fourier lint tuist`
3. See that there are warnings but command completes successfully

### Checklist ✅
